packages/types/src/documents/app/row.ts

```diff
@@ -100,13 +100,17 @@ export enum FieldType {
    */
   BIGINT = "bigint",
   /**
-   * a JSON type, called User within Budibase. This type is used to represent a link to an internal Budibase
+   * an JSON type, called Users within Budibase. It will hold an array of strings. This type is used to represent a link to an internal Budibase
```

Comment:
```suggestion
   * a JSON type, called Users within Budibase. It will hold an array of strings. This type is used to represent a link to an internal Budibase
```